### ISSUE TITLE
[GlobalStep] A crash occurs if tapping ‘update’ multiple times while editing a post.

### ISSUE THREAD

**Issue by wptester9845** [wordpress-mobile/WordPress-iOS#13469]
### Description
If the user selects to edit a post and taps the ‘update’ button in quick succession a crash occurs. This occurs with and without block editor enabled. Due to small differences in the UI, this issue cannot be replicated on Android.

### Reproduction Rate
4/4 100%

### Expected behavior
Once the update button is pressed it will no longer register input until the update has been applied.

### Actual behavior
Tapping update multiple times can cause a crash.

### Steps to reproduce the behaviour
1.	Install WordPress 14.2
2.	Login.
3.	Select to edit an existing post.
4.	Tap update quickly.

##### Tested on the following
iPhone 6 (12.4)
iPhone SE (12.0.1)

#### Please see the attached video for more information
[edit_update_crash.zip](https://github.com/wordpress-mobile/WordPress-iOS/files/4204923/edit_update_crash.zip)

### Submitted by:
Oliver Howe

**Comment by designsimply** [comment]
I was able to replicate this crash with the steps provided, but very inconsistently. For me, I only saw the crash when repeatedly tapping Update in quick succession on draft posts in a Jetpack site. ([1m34s](https://cloudup.com/ctIs6xEM2Gm))

<sup>Tested with WPiOS 14.2.0.2 TestFlight beta on iPhone 6S iOS 13.3.1 using three different types of sites: WordPress.com (madefortesting0318.blog), Self-hosted (Vocational Galliform), Jetpack (Anxious Goat).</sup>

I found my crash report in Sentry in https://sentry.io/share/issue/9c761559d2e143ec8f1ad8710e00d2d9/ and it appears to match up with https://github.com/wordpress-mobile/WordPress-iOS/issues/13430 where there's a PR in progress. 🤞
